data/mapcss-mapping.csv

```diff
@@ -436,7 +436,7 @@ healthcare|sample_collection;435;
 sport|australian_football;436;
 leisure|bowling_alley;437;
 healthcare|speech_therapist;438;
-deprecated|deprecated;439;x
+shop|camera;439;
```

Comment:
Merge with shop=photo or needed a separate type? WDYT?
If separate, please, move to the very end of this file, where other "popular" shops :)

Reply:
Separate, 'photo' is for photo printing/film/framing etc, 'camera' is for the cameras themselves :) will move it 👍